common/src/main/java/com/microsoft/identity/common/internal/eststelemetry/PublicApiId.java

```diff
@@ -59,6 +59,8 @@
     // SingleAccountPublicClientApplication;
     //==============================================================================================
     public static final String SINGLE_ACCOUNT_PCA_SIGN_IN = "123";
+    public static final String SINGLE_ACCOUNT_PCA_SIGN_IN_WITH_PROMPT = "130";
```

Comment:
@shahzaibj Once this PR is approved/merged, we should update the telemetry spreadsheet with this info

Reply:
Yeah, I'll update that. Thanks @iambmelt